extensions/amp-font/0.1/fontloader.js

```diff
@@ -160,13 +160,11 @@ export class FontLoader {
           } else if (this.compareMeasurements_()) {
             resolve();
           } else {
-            vsync.run(vsyncTask);
```

Comment:
do we still want to expose `run` as public if `createTask` is the preferred interface?